extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -989,8 +997,10 @@ async function createRadioOptions(context: FieldContext, getRadioButtonInfo?: ((
 	});
 	addLabelInputPairToContainer(context.view, context.components, label, radioGroupLoadingComponentBuilder.component(), context.fieldInfo);
 	const options = context.fieldInfo.options as OptionsInfo;
-	await radioGroupLoadingComponentBuilder.loadOptions(
-		getRadioButtonInfo || options); // wait for the radioGroup to be fully initialized
+	// Start loading the options but continue on so that we can continue setting up the rest of the components - the group
+	// will show a loading spinner while the options are loaded
+	radioGroupLoadingComponentBuilder.loadOptions(
```

Comment:
I took a look at the implementation of the radioGroupLoadingComponentBuilder and the way we use it here, IIUC, screen reader will announce the default text: loading/loading completed, I think we should provide more meaningful loadingText/loadingCompletedText to the loading component. especially there could be multiple such fields on a page.

Reply:
I'll do a follow up PR with that functionality added